test/Driver/linker.swift

```diff
@@ -108,6 +111,25 @@
 // LINUX-x86_64-DAG: -Xlinker -undefined
 // LINUX-x86_64: -o linker
 
+// LINUX-armv6: swift
+// LINUX-armv6: -o [[OBJECTFILE:.*]]
+
+// LINUX-armv6: clang++{{"? }}
+// LINUX-armv6-DAG: [[OBJECTFILE]]
+// LINUX-armv6-DAG: -lswiftCore
+// LINUX-armv6-DAG: -L [[STDLIB_PATH:[^ ]+/lib/swift]]
+// LINUX-armv6-DAG: --target=junk-unknown-linux-gnueabihf
```

Comment:
While attempting to add testing for whether --target is added to the clang++ invocation, this seems like the correct approach.  However, this does not cause a failure as expected.